pkg/services/alerting/store.go

```diff
@@ -168,6 +168,7 @@ func (ss *sqlStore) HandleAlertsQuery(ctx context.Context, query *alertmodels.Ge
 			builder.Write(")")
 		}
 
+		// FIXME: should not read for orgroles but only permissions
 		if query.User.OrgRole != org.RoleAdmin {
```

Comment:
@IevaVasiljeva we should refactor this right? as we should not check for Roles

Reply:
Yup, you're right. Do you mind creating a follow-up issue for this?

Reply:
https://github.com/grafana/grafana-enterprise/issues/4994 follow up issue